<launch>
    <node type="fake_planner" name="fake_planner" pkg="fake_planner" output="screen">
        <param name="goal_topic" value="move_base_simple/goal"/>
        <param name="cmd_vel_topic" value="/cmd_vel"/>
        <param name="goal_reached_topic" value="/goal_reached"/>
        <param name="max_velocity_service" value="set_max_velocity"/>
        <param name="planner_rate" value="10.0"/>
        <param name="max_linear_speed" value="0.5"/>
        <param name="max_angular_speed" value="0.75"/>
        <param name="position_tolerance" value="0.1"/>
        <param name="orientation_tolerance" value="0.1"/>
        <param name="heading_tolerance" value="0.8"/>
        <param name="enable_orientation_alignment" value="false"/>
        <param name="time_to_pos" value="1"/>
        <param name="time_to_align" value="1"/>
        <param name="time_to_heading" value="1"/>
    </node>
</launch>
